stack/adding-a-new-package.rst

```diff
@@ -59,43 +59,23 @@ With checks enabled people will be able to use the GitHub merge button on Pull R
 .. image:: /_static/build-ci/github_branch_protection.png
 
 In the image above no automated status checks are being performed because we have not yet enabled any.
-GitHub requires that at least one check runs before the up-to-date checks are enabled so a Travis job **must** be provided if the GitHub merge button is to be used.
-Travis does not replace normal testing done with a :doc:`Jenkins job <jenkins-stack-os-matrix>`, but for packages that have been updated to :ref:`use flake8 <testing-flake8>` it is useful to add a simple Travis script like the following:
+GitHub requires that at least one check runs before the up-to-date checks are enabled so a GitHub Action or Travis job **must** be provided if the GitHub merge button is to be used.
+GitHub Actions or Travis do not replace normal testing done with a :doc:`Jenkins job <jenkins-stack-os-matrix>`.
+For packages that contain Python, it is useful to add a simple GitHub Action by selecting "Actions" from the GitHub repository page, selecting "New Workflow" if necessary, and choosing the "LSST DM Python lint Workflow".
+If Python typing is used, it can be checked using ``mypy`` via the "LSST DM Python mypy Workflow".
+Similarly, YAML files can be checked via the "LSST DM YAML lint Workflow", and shell scripts can be checked via the "LSST DM shellcheck Workflow".
```

Comment:
May want to have a parenthetical noting that this may require defining a yamllint configuration file.